lib/engine/graph.rb

```diff
@@ -89,6 +89,69 @@ def reachable_hexes(corporation)
       @reachable_hexes[corporation]
     end
 
+    # How far is the start_hex from one of the corporation's stations?
+    # This is by track, not as the crow flies.
+    # start_cy is either a City or nil (plain track or town)
+    #
+    # This implementation assumes:
+    #  - Lawson Track
+    #  - No double dits
+    #  - OO tiles should work? NY tile works
+    def distance_to_station(corporation, start)
+      goal_hexes = corporation.tokens.select(&:city).map { |t| [t.city.hex, t.city] }
+      distance = 0
+      visited_hexes = []
+      start_hexes = if start.respond_to?(&:city?) && start.city?
```

Comment:
why? 

what is start?

can't you just do start.is_a?(City)

Reply:
`start` is where to start searching from (the city or hex to be tokened {PlaceToken or HexToken))

I figured using .city? would be more canonical but I went and changed it to `.is_a?(Engine::Part::City)`